R/zzz.R

```diff
@@ -171,7 +171,9 @@ all_undesirable_functions <- modify_defaults(
   untrace = paste(
     "remove this likely leftover from debugging.",
     "It is only useful for interactive debugging with trace()"
-  )
+  ),
+  structure =
```

Comment:
Is it possible to do a trailing comma trick here, too?

Reply:
not trivially -- rlang::list2() inside modify_defaults() would be the right approach to let someone else handle the implementation, or we'd have to hack it together ourselves looking for missing() at the end of ...

Reply:
Okay, never mind then.